### ISSUE TITLE
Possible fix for better font rendering

### ISSUE THREAD

**Issue by l2-** [runelite/runelite#1278]
This isn't an issue but i think i found out why a lot of fonts look slightly off (especially bolded)

`Toolkit tk = Toolkit.getDefaultToolkit();
Map desktopHints = (Map)(tk.getDesktopProperty("awt.font.desktophints"));
if (desktopHints != null)
{
	graphics.addRenderingHints(desktopHints);
}`

for testing i placed this inside the render void in overlayrenderer but i don't know if that's the good place or if you guys care about this at all, hence an issue instead of pr.
From my testing i found that the regular runescape font bolded doesn't look like garbage anymore with this setting.

**Comment by deathbeam** [comment]
What about `-Dawt.useSystemAASettings`? https://docs.oracle.com/javase/8/docs/technotes/guides/2d/flags.html#aaFonts

I feel like that might work too, but I am not sure.

**Comment by l2-** [comment]
running with -Dawt.useSystemAASettings doesn't seem to work for me

**Comment by Nightfirecat** [comment]
Is this still an issue? I can't say I know what "a lot of fonts looking slightly off" looks like. I've never experienced the fonts looking strange on my end on any machine/environment.

**Comment by l2-** [comment]
I don't think you can see the difference with fonts such as the runescape fonts since they don't show aliasing artifacts. I only noticed when using system fonts with https://github.com/runelite/runelite/pull/6130
